flutter_theoplayer_sdk/lib/widget/fullscreen_widget.dart

```diff
@@ -38,7 +53,8 @@ class _FullscreenStatelessWidgetState extends State<FullscreenStatelessWidget> {
             return Center(
                 // Center is a layout widget. It takes a single child and positions it
                 // in the middle of the parent.
-                child: !willPop ? ChromelessPlayer(key: ChromelessPlayer.globalKey, player: widget.theoplayer) : Container()
+                child:
+                  !willPop ? Container(color: Colors.black, child: PresentationModeAwareWidget(player: widget.theoplayer, presentationModeToCheck: const [PresentationMode.FULLSCREEN],)) : Container(),
```

Comment:
I would leave it like this, because `!willPop` means we need to render the player (so we are not in the closing moment of the screen)

((Maybe renaming or extracting it to an other variable could make clearer (e.g. `shouldShow = !willPop`)))